wavepool/templates/wavepool/base.html

```diff
@@ -3,7 +3,7 @@
 	<head>
   		<link href="{% static 'bootstrap/css/bootstrap.min.css' %}" rel="stylesheet">
   		<link href="{% static 'app.css' %}" rel="stylesheet">
-  		<title>Wavepool | Industry Dive</title>
+  		<title>{{ newspost.title }} | Wavepool | Industry Dive</title>
```

Comment:
This fixes was made to include the `newspost.title` in the template as required by  `test_newspost_page_title_attribute`